nservicebus/containers/index_content_core_[,5].partial.md

```diff
@@ -5,6 +5,7 @@ Note that the instance of `IBus` is scoped for the lifetime of the container. He
 
 ### Cleanup
 
-When using an external container normally the bus instance is not disposed of manually. If `IBus.Dispose()` is called that would indirectly trigger the container to dispose lifetime scope.
+When using an external container normally the bus instance is not disposed of automatically. The following dispose sequence should be applied if latest container adapter packages are used:
```

Comment:
@danielmarbach - yeah, i get it. it is tricky! how about the below? 

When using an external container, the bus instance is not automatically disposed. In order to dispose the resources properly:
1. Dispose the bus by calling `IBus.Dispose()`.
2. Dispose the external container with `container.Dispose()`

This behavior was introduced from Versions 6 and above of NServiceBus and also back-ported to Version 5 of NServiceBus. Therefore, ensure that the version of NServiceBus used in the project supports this behavior when making the suggested code change.